lib/peeps.rb

```diff
@@ -0,0 +1,37 @@
+require 'pg'
+
+class Peep
+  
+  attr_reader :id, :username, :message
+
+  def initialize(id:, username:, message:)
+    @id = id
+    @username = username
+    @message = message
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
```

Comment:
I think these ENV tests could be moved to another file or even just another method to keep everything DRY!